### ISSUE TITLE
[Question] is posible to send a message through the ipc from the APP to the DLL?

### ISSUE THREAD

**Issue by Raagh** [EasyHook/EasyHook#172]
im hooking with no issues and sending messages(through the interface) from the DLL to the console application(everything in C#) with no issues at all. 

Is there a way to send a message from the Application to DLL?

Example: i want that some events in my application changes configurations in the dll.

**Comment by spazzarama** [comment]
Yes, you can see an example of this happening here: https://github.com/spazzarama/Direct3DHook/blob/master/Capture/EntryPoint.cs#L33
